File: implementation/scipamato/scipamato-public-web/src/main/java/ch/difty/scipamato/publ/web/newstudies/NewStudyTopic.java
package ch.difty.scipamato.publ.web.newstudies;

import java.io.Serializable;
import java.util.List;

import lombok.Value;

@Value
public class NewStudyTopic implements Serializable {

    private static final long serialVersionUID = 1L;

    private final int            sort;
    private final String         title;
    private final List<NewStudy> studies;

}

File: public/public-entity/src/main/kotlin/ch/difty/scipamato/publ/entity/NewStudyTopic.kt
package ch.difty.scipamato.publ.entity

data class NewStudyTopic(
    val sort: Int = 0,
    val title: String? = null,
    val studies: List<NewStudy> = emptyList(),
) : PublicDbEntity {
    companion object {
        private const val serialVersionUID = 1L
    }
}
